src/utils.rs

```diff
@@ -4,8 +4,6 @@ use slog_term;
 
 use lazy_static::lazy_static;
 
-pub type BoxResult<T> = Result<T, Box<std::error::Error>>;
```

Comment:
Not really needed, Boxing just added meaningless overhead and hid some meaningful matter.

Now that main returns Postgres Error it would likely also provide better Error reporting(I am guessing here).